front/cypress/e2e/auth/sign_up_custom_fields.cy.ts

```diff
@@ -1,6 +1,26 @@
 import { randomString, randomEmail, logout } from '../../support/commands';
 
 describe('Sign up - custom fields step', () => {
+  describe('No custom fields', () => {
+    before(() => {
+      const firstName = randomString();
+      const lastName = randomString();
+      const email = randomEmail();
+      const password = randomString();
+      cy.apiSignup(firstName, lastName, email, password);
+      cy.setLoginCookie(email, password);
+    });
+
+    after(() => {
+      logout();
+    });
+
+    it('does not show it when no custom fields are enabled', () => {
+      cy.goToLandingPage();
+      cy.get('#e2e-sign-up-in-modal').should('not.exist');
+    });
+  });
```

Comment:
Think I might have deleted this when I moved this test to the `auth` folder. Put it back and move the `goToLandingPage` call into the test (I think there might be an issue with assertions and waiting for stuff to be available in `before` blocks that's handled better in tests)